packages/https-proxy/lib/ca.coffee

```diff
@@ -135,7 +135,7 @@ class CA
     sn
 
   generateCA: ->
```

Comment:
we can do this on cypress start since it could be a lil slow

Reply:
well, it's already done on server start here:

https://github.com/cypress-io/cypress/blob/e4cd4df1e8389ad5ae2c5aea13a079358055fa68/packages/server/lib/server.coffee#L266-L272